src/Compilers/VisualBasic/Portable/BoundTree/BoundPropertyAccess.vb

```diff
@@ -13,7 +13,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
     End Enum
 
     Partial Friend Class BoundPropertyAccess
-        Public Sub New(syntax As SyntaxNode, propertySymbol As PropertySymbol, propertyGroupOpt As BoundPropertyGroup, accessKind As PropertyAccessKind, isWriteable As Boolean, receiverOpt As BoundExpression, arguments As ImmutableArray(Of BoundExpression), Optional hasErrors As Boolean = False)
+        Public Sub New(syntax As SyntaxNode, propertySymbol As PropertySymbol, propertyGroupOpt As BoundPropertyGroup, accessKind As PropertyAccessKind, isWriteable As Boolean, isLValue As Boolean, receiverOpt As BoundExpression, arguments As ImmutableArray(Of BoundExpression), type As TypeSymbol, Optional hasErrors As Boolean = False)
```

Comment:
>Public Sub New(s [](start = 8, length = 16)

Please break this long line

Reply:
Going to remove this constructor, no longer relevant.

---
In reply to: [190019739](https://github.com/dotnet/roslyn/pull/26781#discussion_r190019739) [](ancestors = 190019739)